ThirdParty/GNUmakefile

```diff
@@ -0,0 +1,95 @@
+#
+# Build configuration
+#
+
+COMP            = gcc
+NVCC            = nvcc
+DEBUG           = FALSE
+USE_CUDA        = FALSE
+USE_KLU         = FALSE
+
+#SS_VERSION=5.7.2
+SS_VERSION=5.4.0
+
+#include $(AMREX_HOME)/Tools/GNUMake/Make.defs
+
+tp_suffix=$(COMP)$(DebugSuffix)$(CUDASuffix)$(KLUSuffix)
+INSTALL_PREFIX=$(PWD)/INSTALL/$(tp_suffix)
+
+ifeq ($(USE_KLU),TRUE)
```

Comment:
I don't understand if suite sparse is build somewhere in there, if does the user have to do it before ?

Reply:
Ok never mind I god this one